docs/details/commands/pmm-admin.md

```diff
@@ -12,6 +12,8 @@
 
 `pmm-admin add DATABASE [FLAGS] [NAME] [ADDRESS]`
 
+`pmm-admin add external [FLAGS] [NAME] [ADDRESS]`
```

Comment:
This is still in Tech Preview, not GA ready feature. let's  mention this to avoid confusion